tools/runqlen.bt

```diff
@@ -5,6 +5,8 @@
  *
  * This is a bpftrace version of the bcc tool of the same name.
  *
+ * For Linux 5.7+ (see tools/old for script for lower versions).
```

Comment:
Technically, this is not correct. If BTF is available, the correct definition of `struct cfs_rq` will be pulled.

That being said, I'm ok with this change, I just wouldn't mention that the script is for kernel 5.7+ only. Also I'd avoid keeping the old version of the tool as the comment below explicitly states that the struct def may require user's intervention (on systems without BTF).

Reply:
@viktormalik Thanks, I have made the requested changes, this will be just to ensure that the script remains compatible with the most recent kernel version.

Reply:
Thanks! 

IMHO, any recent kernel version *should* have BTF enabled but I understand that may not always be the case.